lib/app/components/molecules/gender_select_checkbox.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:favoritism_communication/app/services/gender_service.dart';
+import 'package:favoritism_communication/app/components/atoms/custom_checkbox.dart';
+import 'package:get/get.dart';
+
+class GenderSelectCheckbox extends StatelessWidget{
+  const GenderSelectCheckbox({
+    super.key,
+    this.onChanged,
+    this.values,
+  });
+
+  final Map<Gender, bool>? values;
+  final void Function(Gender? gender, bool? value)? onChanged;
+
+  @override
+  Widget build(BuildContext context){
+    return Row(
+      children: Gender.values.map((gender){
+        return SizedBox(
+          width: context.width / Gender.values.length,
+          child: CustomCheckbox(
+            label: gender.label,
+            value: values?[gender] ?? false,
+            onChanged: (value) => onChanged?.call(gender, value),
+          ),
+        );
+      }).toList(),
+    );
```

Comment:
```suggestion
    return Column(
      crossAxisAlignment: CrossAxisAlignment.start,
      children: [
        const Padding(
          padding: EdgeInsets.only(top: 8, left: 8, right: 8),
          child: Text(
            '性別',
            textAlign: TextAlign.left,
            style: TextStyle(
              fontWeight: FontWeight.bold,
            ),
          ),
        ),
        Row(
          children: Gender.values.map((gender) {
            return SizedBox(
              width: Get.width / 4,
              child: _CustomCheckbox(
                label: gender.label,
                value: values?[gender] ?? false,
                onChanged: (value) => onChanged?.call(gender, value),
              ),
            );
          }).toList(),
        ),
      ],
    );
```
「性別」の項目名がないので追加お願いします。
あとフォントサイズを微修正しないとオーバーフローしちゃうのでついでに修正してください
（suggestionにオーバーフローを修正した内容も含めてます）